util/internal/integrationtest/mapper/orm/src/main/java/org/hibernate/search/util/impl/integrationtest/mapper/orm/ReusableOrmSetupHolder.java

```diff
@@ -500,7 +500,11 @@ private static void clearEntityInstances(SessionFactory sessionFactory, Hibernat
 	}
 
 	private static Query<?> createSelectOrDeleteAllOfSpecificTypeQuery(EntityType<?> entityType, Session session, String prefix) {
-		StringBuilder builder = new StringBuilder( prefix ).append( "from " ).append( entityType.getName() ).append( " e" );
+		StringBuilder builder = ( prefix == null || prefix.trim().isEmpty() ) ?
```

Comment:
Please use `org.hibernate.search.util.common.impl.StringHelper` when you can:

```suggestion
		StringBuilder builder = StringHelper.isEmpty( prefix ) ?
```

Though to be honest, I don't think the prefix is ever null here. The old code wouldn't have worked with a `null` prefix.

Reply:
I guess you disagree :)

Reply:
Actually I miss it :( My bad